includes/Checker/CLI_Runner.php

```diff
@@ -37,7 +37,7 @@ public function is_plugin_check() {
 		}
 
 		if (
-			'wp' === $_SERVER['argv'][0] &&
+			'wp' === substr( $_SERVER['argv'][0], -2 ) &&
 			'plugin' === $_SERVER['argv'][1] &&
 			'check' === $_SERVER['argv'][2]
 		) {
```

Comment:
This code seems to be repeated. It worth moving this to a util.

Reply:
Thanks @spacedmonkey Yes, we've noticed this too and other elements of the codebase as well that is repeated. See my other comment https://github.com/10up/plugin-check/pull/100#discussion_r1146032399 I think it's best to tackle some of this refactoring as part of next weeks review once the wider architecture is complete.

Reply:
We should fix this later, add a comment / todo in the code to fix.

Reply:
@spacedmonkey we have not done this with other areas we would like to refactor and I don't think adding comments now will add any value. We'll be reviewing items like this as part of the wider review, for now we want to be reviewing and addressing anything critical.

Reply:
See https://github.com/10up/plugin-check/issues/111